{"name": "test_login_failed[case2]", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('init', <SubRequest 'back_login' for <Function test_login_failed[case2]>>)", "trace": "file D:\\Test_framework\\Web_auto\\TestCases\\test_login1.py, line 11\n      @pytest.mark.parametrize(\"case\", td.invalid_data)\n      def test_login_failed(self, case, back_login):\nfile D:\\Test_framework\\Web_auto\\TestCases\\conftest.py, line 38\n  @pytest.fixture(scope=\"class\")\n  def back_login(init):\nE       fixture 'init' not found\n>       available fixtures: back_login, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, login, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nD:\\Test_framework\\Web_auto\\TestCases\\conftest.py:38"}, "parameters": [{"name": "case", "value": "{'user': '1868472055', 'passwd': 'python', 'check': '帐号或密码错'}"}], "start": 1647849780712, "stop": 1647849780712, "uuid": "93f19d48-1b5a-4842-9376-d59fe5771173", "historyId": "1bd1937941fff8809d36a7e206d429da", "testCaseId": "6d230ca66b6dfc3f741b1dec342650f6", "fullName": "TestCases.test_login1.TestLogin#test_login_failed", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('back_login')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login1"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "MS-AWORNJEVXPKP"}, {"name": "thread", "value": "19376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login1"}]}